test/jbuild

```diff
@@ -0,0 +1,9 @@
+(executable
+ ((name test)
+  (libraries (ocaml-netlink))
+  ))
+  
+(alias
+ ((name   runtest)
+  (deps   (test.exe))
+  (action (run ${<} -runner sequential))))
```

Comment:
I think you don't need `-runner sequential`, it was not specified in the oasis file